contracts/src/v0.8/dev/automation/2_1/KeeperRegistryBase2_1.sol

```diff
@@ -235,6 +369,10 @@ abstract contract KeeperRegistryBase2_1 is ConfirmedOwner, ExecutionPrevention {
     uint8 index;
   }
 
+  struct BlockTriggerConfig {
+    uint32 checkCadance; // how often to check in blocks
+  }
+
```

Comment:
I'm not sure why we need this strictly on the contract

Reply:
I just added it cor consistency with the other getters. I think yes let's try to use the type-agnostic getter. In the short term, I'm indifferent though.